deployment_scripts/README.md

```diff
@@ -1,25 +1,49 @@
-# Files used to deploy development version of infection monkey
-## Windows
-
-Before running the script you must have git installed.<br>
-Cd to scripts directory and use the scripts.<br>
-First argument is an empty directory (script can create one) and second is branch you want to clone.
-Example usages:<br>
-./run_script.bat (Sets up monkey in current directory under .\infection_monkey)<br>
-./run_script.bat "C:\test" (Sets up monkey in C:\test)<br>
-powershell -ExecutionPolicy ByPass -Command ". .\deploy_windows.ps1; Deploy-Windows -monkey_home C:\test" (Same as above)<br>
-./run_script.bat "" "master"(Sets up master branch instead of develop in current dir)
-Don't forget to add python to PATH or do so while installing it via this script.<br>
-
-## Linux
-
-Linux deployment script is meant for Ubuntu 16.x machines.
-You must have root permissions, but don't run the script as root.<br>
-Launch deploy_linux.sh from scripts directory.<br>
-First argument should be an absolute path of an empty directory (script will create one if doesn't exist, default is ./infection_monkey).
-Second parameter is the branch you want to clone (develop by default).
-Example usages:<br>
-./deploy_linux.sh (deploys under ./infection_monkey)<br>
-./deploy_linux.sh "/home/test/monkey" (deploys under /home/test/monkey)<br>
-./deploy_linux.sh "" "master" (deploys master branch in script directory)<br>
-./deploy_linux.sh "/home/user/new" "master" (if directory "new" is not found creates it and clones master branch into it)<br>
+# Deployment guide for a development environemnt
+
+This guide is for you if you wish to develop for Infection Monkey. If you only want to use it, please download the relevant version from [our website](https://infectionmonkey.com).
+
+## Prerequisites
+
+Before running the script you must have `git` installed. If you don't have `git` installed, please follow [this guide](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+## Deploy on Windows
+
+Run the following command in powershell:
+
+```powershell
+Invoke-WebRequest https://raw.githubusercontent.com/guardicore/monkey/develop/deployment_scripts/deploy_windows.ps1 -OutFile deploy_windows.ps1
+```
+
+This will download our deploy script. It's a good idea to read it quickly before executing it!
+
+After downloading that script, execute it in a shell (like `cmd` or `powershell`). The first argument is an empty directory (script can create one). The second argument is which branch you want to clone - by default, the script will check out the `develop` branch. Some example usages:
+
+- `.\deploy_windows.ps1` (Sets up monkey in current directory under .\infection_monkey)
```

Comment:
You sure you dont need `powershell` at the start? For me it opens .\deploy_windows.ps1 file with notepad.

Reply:
What version of PS are you executing that command in?
I see I need to fix a mistake anyway, developers shouldn't be executing anything in the `cmd` shell